platforms/web/src/App.tsx

```diff
@@ -186,6 +187,19 @@ function App() {
                         <button type="button" onClick={(_e) => wysiwyg.clear()}>
                             clear
                         </button>
+                        {suggestion && (
+                            <button
+                                type="button"
+                                onClick={(_e) =>
+                                    wysiwyg.mention(
+                                        'https://matrix.to/#/@a_user:element.io',
+                                        'a user',
```

Comment:
nitpick: `a user` looks a bit jarring IMO - maybe we could we go with a 'fake' username e.g. `@alice`?

Reply:
Updated